pkg/controller/controller_test.go

```diff
@@ -968,7 +968,8 @@ func TestReconcileInstanceDelete(t *testing.T) {
 	actions := filterActions(fakeCatalogClient.Actions())
 	// The two actions should be:
 	// 0. Updating the ready condition
-	// 1. Removing the finalizer
+	// 1. Get against the instance
+	// 2. Removing the finalizer
```

Comment:
ok, why do we say '2' but then expect 3?  And let's start this human readable list from 1.